.github/workflows/itest-deploy.yml

```diff
@@ -155,15 +114,27 @@ jobs:
       - name: Run ideploy (Deploy to the environment)
         run: |
           # Update comment with Pending status for this step
```

Comment:
This syntax seems incorrect. You are starting a multiline block for the `run:` key, but then add a comment and another yaml key (`uses`) so this will not function as expected.